src/QmlControls/QGCPipable.qml

```diff
@@ -188,4 +189,37 @@ Item {
             }
         }
     }
+    Rectangle {
+        id:                     buttonScreenShoot
+        anchors.left:           openPIP.right
+        anchors.bottom:         parent.bottom
+        height:                 ScreenTools.defaultFontPixelHeight * 2
+        width:                  ScreenTools.defaultFontPixelHeight * 2
+        radius:                 ScreenTools.defaultFontPixelHeight / 3
+        visible:                true
+        color:                  isDark ? Qt.rgba(0,0,0,0.75) : Qt.rgba(0,0,0,0.5)
+        focus: visible
```

Comment:
this seems like it could interfere with other items that might want focus, is it right?

Reply:
yes, it's right. we need focus *or* a customized focus handler, to trigger actions with the keyboard.